release-notes/release-5.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: April 17, 2024
+---
+
+# Features
+
+- [Oracle Database Integration]() with default dashboard
+- [Apache Networking Integration]() with default dashboard
+- Automated log collection with the Vercel APM for __Build__, __Lambda__, __Static__, __Edge__, and __External__ immediately upon installation
+- Ability to parse attributes from JSON log messages
+- Can use email addresses as query parameters within RUM to easily share user-specific sessions with your teammates
```

Comment:
"Improved sharing of RUM links with filters" since it's all filters, email was just the called out one